Formula/sifu.rb

```diff
@@ -0,0 +1,22 @@
+class Sifu < Formula
+  desc "Rapid software development tool made for creating cloud solutions."
```

Comment:
We cannot include this in Homebrew if it's not open source, sorry. Homebrew Cask may accept it instead. Please also consider [creating your own tap](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap) instead. Thanks!